imap_processing/lo/l0/lol0.py

```diff
@@ -0,0 +1,48 @@
+"""General Lo L0 data class used for parsing data and setting attributes."""
+from dataclasses import dataclass, fields
+
+from imap_processing.ccsds.ccsds_data import CcsdsData
+
+
+@dataclass
+class LoL0:
+    """Data structure for common values across histogram and direct events data.
```

Comment:
Does Lo also have histogram data? Or is this leftover from GLOWS?

Reply:
Good catch. They do, but not here. That's there because I copied your GLOWS code.